imap_processing/cdf/config/imap_ultra_l1c_variable_attrs.yaml

```diff
@@ -134,3 +134,10 @@ packetdata:
   LABLAXIS: packet data
   # TODO: come back to format
   UNITS: " "
+
+energy_bin_delta:
```

Comment:
Will this have a dimension in the xarray? (I think the CDF-speak would be a Depend) 

I'm expecting it to have one dimension `dims = ("energy_bin_geometric_mean",)`